spec/actors/sufia/create_with_files_actor_spec.rb

```diff
@@ -44,4 +44,28 @@
       end
     end
   end
+
+  context "with mediated deposit" do
+    let(:inactive) { ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive') }
+    context "flipped on" do
```

Comment:
Could you apply a small refactoring to DRY the new specs out a little? Here's what I had in mind

```ruby
describe "mediated deposit" do
  subject { actor.curation_concern.state }
  let(:inactive_uri) { RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive') }
  before do
    allow(FlipFlop).to receive(:enable_mediated_deposit?).and_return(mediated_enabled)
    allow(AttachFilesToWorkJob).to receive(:perform_later).with(GenericWork, [uploaded_file1, uploaded_file2])
    actor.create(attributes)
  end
  context "when enabled" do
    let(:mediation_enabled) { true }
    it { is_expected.to eq inactive_uri }
  end
  context "when disabled" do
    let(:mediation_enabled) { false }
    it { is_expected.to be nil }
  end
end
```